dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/CacheHashMap.java

```diff
@@ -1134,14 +1169,21 @@ protected BackedSession readFromExternal(String id) {
         final boolean trace = TraceComponent.isAnyTracingEnabled();
 
         CacheSession sess = null;
+        ArrayList<?> value = null;
 
-        if (trace && tc.isDebugEnabled())
-            tcInvoke(tcSessionMetaCache, "get", id);
+        try {
+            if (trace && tc.isDebugEnabled())
+                tcInvoke(tcSessionMetaCache, "get", id);
 
-        ArrayList<?> value = sessionMetaCache.get(id);
+            value = sessionMetaCache.get(id);
 
-        if (trace && tc.isDebugEnabled())
-            tcReturn(tcSessionMetaCache, "get", value);
+            if (trace && tc.isDebugEnabled())
+                tcReturn(tcSessionMetaCache, "get", value);
+        } catch(Exception ex) {
+            FFDCFilter.processException(ex, "com.ibm.ws.session.store.cache.CacheHashMap.removePersistedSession", "1156", id);
```

Comment:
In a number of places, not just this one, I see a different value being supplied to FFDCFilter in place of what is usually the 'this' object for the callerThis parameter.  There is a signature of the method that allows for both.